e2e-tests/tls-issue-cert-manager/conf/some-name.yml

```diff
@@ -0,0 +1,84 @@
+apiVersion: pxc.percona.com/v1
```

Comment:
why we need this file? can we use just default `$conf_dir/$cluster.yml`?

Reply:
i'm using it because i add TLS section with additional `SAN's`